addons/sitting/functions/fnc_canSit.sqf

```diff
@@ -21,10 +21,9 @@ params ["_seat", "_player", ["_seatPos", 0]];
 
 // Sitting enabled, not occupied and standing up (or not on a big slope)
 XGVAR(enable) &&
-{[_player, _seat] call EFUNC(common,canInteractWith)} &&
```

Comment:
we need this or handcuffed people can sit down
or sit on chairs being carried (I think)

Reply:
Good point.
The problematic line for #10486 is that line unfortunately. It checks for the owner, which is the root cause of the issue.
Either we extract `EFUNC(common,canInteractWith)`'s content without the owner checking part (so just check for the conditions), temporarily set the owner variable to user interacting (could be done with local effect only, but jank af as a solution) or add a parameter to `EFUNC(common,canInteractWith)` to not check for the owner (imo the most adequate solution).

Do you have any other ideas?